include/grpc++/time.h

```diff
@@ -34,6 +34,48 @@
 #ifndef GRPC_INTERNAL_CPP_UTIL_TIME_H
 #define GRPC_INTERNAL_CPP_UTIL_TIME_H
 
+#include <grpc++/config.h>
+
+namespace grpc {
+
+/* If you are trying to use CompletionQueue::AsyncNext with a time class that
+   isn't either gpr_timespec or std::chrono::system_clock::time_point, you
+   will most likely be looking at that comment as your compiler will have
+   fired an error below. In order to fix that issue, you have two potential
+   solutions:
```

Comment:
Minor nit: most of the "that" above should be "this" -- "this comment" ,  "this issue"